commons-numbers-core/src/main/java/org/apache/commons/numbers/core/ArithmeticUtils.java

```diff
@@ -266,6 +266,16 @@ public static long lcm(long a, long b) {
     /**
      * Raise an int to an int power.
      *
+     * <p>Special cases:</p>
```

Comment:
You should put the same comment on the `long` version of the pow method.

Reply:
@aherbert Ah, I forgot it :)
will do now.